web/src/app/modules/shared/components/presentation/datagrid/datagrid.component.html

```diff
@@ -46,9 +46,20 @@ <h4 class="clr-col-10" *ngIf="showTitle()">{{ title }}</h4>
     <clr-dg-cell *ngFor="let column of columns; trackBy: identifyColumn">
       <app-view-container [view]="row.data[column]"></app-view-container>
     </clr-dg-cell>
-    <ng-container ngProjectAs="clr-dg-row-detail" *ngIf="row?.expandedDetail">
+    <!-- expandedDetails-->
```

Comment:
None of these comments contribute to the readability or context of this template.